soc/nxp/imxrt/imxrt10xx/Kconfig

```diff
@@ -89,7 +89,7 @@ if PM
 config DCDC_TARGET_LOW_POWER_VOLTAGE
 	int "Target DCDC voltage in mV for low power mode"
 	default 1075
-	range 800 1575
+	range 925 1300
```

Comment:
Should we limit the range, or just add a comment in the `help` description?

Reply:
I think better to limit the range, other value is not guaranteed.